graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/CodeGen.kt

```diff
@@ -319,6 +326,52 @@ class CodeGen(private val config: CodeGenConfig) {
             }
             .fold(CodeGenResult()) { t: CodeGenResult, u: CodeGenResult -> t.merge(u) }
     }
+
+    private fun generateKotlin2ForSchema(schema: String): CodeGenResult {
+
+        val parser = Parser()
+        val options = ParserOptions
+            .getDefaultParserOptions()
+            .transform { o -> o.maxTokens(MAX_VALUE) }
+
+        // TODO where should this go?
+        val pageInfo = """
```

Comment:
One of the biggest outstanding issues is handling of implicit/scalar types. For this one, I don't want to redirect to an existing impl because then I'd have to maintain that impl to make sure it matches what we generate. I'd prefer to have it generated along with all of the rest of the schema